jenkins/spark-nightly-build.sh

```diff
@@ -41,7 +41,7 @@ DIST_POM_FPATH="$DIST_PL/target/extra-resources/META-INF/maven/$ART_GROUP_ID/$AR
 DIST_PROFILE_OPT=-Dincluded_buildvers=$(IFS=,; echo "${SPARK_SHIM_VERSIONS[*]}")
 DIST_INCLUDES_DATABRICKS=${DIST_INCLUDES_DATABRICKS:-"true"}
 if [[ "$DIST_INCLUDES_DATABRICKS" == "true" ]]; then
-    DIST_PROFILE_OPT="$DIST_PROFILE_OPT,312db,321db"
+    DIST_PROFILE_OPT="$DIST_PROFILE_OPT,312db,321db,330db"
```

Comment:
nit: could generate the list from pom file as our single source in version-def script

Reply:
Good point
updated